CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### NEXT (unreleased)
 
+### 4.0.0 2017-06-26
```

Comment:
Are there breaking changes? If not, I'd say that `3.7` would be a fine version #.